src/components/table/examples/table-selectable-rows.tsx

```diff
@@ -60,19 +65,25 @@ export class TableExampleSelectableRows {
                 onClick={this.clearSelection}
             />,
             <limel-button label="Reset" onClick={this.resetSelection} />,
+            <limel-switch
```

Comment:
This is really good for testing purposes but is it really relevant for showing the select feature? 🤔

Reply:
I removed it now but the reason it was relevant is because we add a column behind the scenes and the column change event doesn't state that you won't get a column for the row selector. But it may not be necessary to explain that the event will only emit the contents of the columns prop (it may be in another order but never have other items that what is put in the prop)

Reply:
If the row selector is a column we add behind the scenes, I think it's (probably) something that should stay behind the scenes, and thus be filtered out from change-events and the like. (General thoughts; I haven't looked at the specifics in this case.)